builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -294,13 +298,9 @@ def archiveAssetDetails(env) {
  */
 def checkBucket(stage) {
 	withCredentials([
-		[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: configLoader.AWS_CREDENTIAL_ID, secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']
+		[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: service_config.credentialId]
 	]) {
 		try {
```

Comment:
region would still be required to be set in environment variables. same in below too...

Reply:
Regions are passed along with the cli commands.